### PR TITLE
Add i18n to the document object

### PR DESCRIPTION
**Pull request by levimykel**
Adds a new i18n and RelatedDocument class.
Adds the i18n object to the document object.
Adds a SearchForm method to set the language of a query.
Adds i18n testing to the kit

### CODE REVIEW

File: src/Prismic/i18n/i18n.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+namespace Prismic\i18n;
+
+use Prismic\i18n\RelatedDocument;
+
+/**
+ * This class embodies an i18n document object from the prismic.io API,
+ * containing the language of the document, as well as all of the related
+ * documents.
+ */
+class i18n
```
Comment (dohzya) [inline src/Prismic/i18n/i18n.php]: Shouldn’t the class start with a uppercase letter?

Comment (levimykel) [inline src/Prismic/i18n/i18n.php]: Yes, yes it should.

**Comment by levimykel** [comment]
Test are now added!

File: tests/Prismic/I18nTest.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+     * @depends testGetI18n
+     */
+    public function testGetRelatedDoc(Array $i18nArray)
+    {
+        $relatedDocument = $i18nArray[0]->getRelatedDoc('en-us');
+        $relatedDocumentWithUid = $i18nArray[1]->getRelatedDoc('fr-fr');
+        $this->assertInstanceOf('Prismic\I18n\RelatedDocument', $relatedDocument);
+        $this->assertInstanceOf('Prismic\I18n\RelatedDocument', $relatedDocumentWithUid);
+        
+        return array($relatedDocument, $relatedDocumentWithUid);
```
Comment (dohzya) [inline tests/Prismic/I18nTest.php]: Why do you return an array? 🤔

**Pull request closed without merging**